diff --git a/arch/arm/boot/dts/dra7xx-vision.dtsi b/arch/arm/boot/dts/dra7xx-vision.dtsi
index edf14a9..8e232d6 100644
--- a/arch/arm/boot/dts/dra7xx-vision.dtsi
+++ b/arch/arm/boot/dts/dra7xx-vision.dtsi
@@ -91,7 +91,8 @@
 	};

 	lvds_des1: deserializer@60 {
-		compatible = "ti,ds90ub914aq";
+		compatible = "ti,crapub914aq";
+		status = "disabled";
 		reg = <0x60>;

 		gpios =	<&mcp_ioexp1 7 0>;	/* pdb */
@@ -101,7 +102,8 @@
 	};

 	lvds_des2: deserializer@64 {
-		compatible = "ti,ds90ub914aq";
+		compatible = "ti,crapub914aq";
+		status = "disabled";
 		reg = <0x64>;

 		gpios =	<&mcp_ioexp1 15 0>;	/* pdb */
@@ -111,7 +113,8 @@
 	};

 	lvds_des3: deserializer@68 {
-		compatible = "ti,ds90ub914aq";
+		compatible = "ti,crapub914aq";
+		status = "disabled";
 		reg = <0x68>;

 		gpios =	<&mcp_ioexp2 7 0>;	/* pdb */
@@ -121,8 +124,9 @@
 	};

 	lvds_des4: deserializer@6c {
-		compatible = "ti,ds90ub914aq";
+		compatible = "ti,crapub914aq";
 		reg = <0x6c>;
+		status = "disabled";

 		gpios =	<&mcp_ioexp2 15 0>;	/* pdb */

@@ -131,8 +135,9 @@
 	};

 	lvds_des5: deserializer@61 {
-		compatible = "ti,ds90ub914aq";
+		compatible = "ti,crapub914aq";
 		reg = <0x61>;
+		status = "disabled";

 		gpios =	<&mcp_ioexp3 7 0>;	/* pdb */

@@ -141,7 +146,8 @@
 	};

 	lvds_des6: deserializer@69 {
-		compatible = "ti,ds90ub914aq";
+		compatible = "ti,crapub914aq";
+		status = "disabled";
 		reg = <0x69>;

 		gpios =	<&mcp_ioexp3 15 0>;	/* pdb */
@@ -156,7 +162,7 @@
 		 <0x30 0x38>;

 	lvds_ser1: serializer@74 {
-		compatible = "ti,ds90ub913aq";
+		compatible = "ti,crapub913aq";
 		reg = <0x58>;
 		slave-mode;
 	};
@@ -180,7 +186,7 @@
 		 <0x30 0x39>;

 	lvds_ser2: serializer@75 {
-		compatible = "ti,ds90ub913aq";
+		compatible = "ti,crapub913aq";
 		reg = <0x58>;
 		slave-mode;
 	};
@@ -204,7 +210,7 @@
 		 <0x30 0x3a>;

 	lvds_ser3: serializer@76 {
-		compatible = "ti,ds90ub913aq";
+		compatible = "ti,crapub913aq";
 		reg = <0x58>;
 		slave-mode;
 	};
@@ -228,7 +234,7 @@
 		 <0x30 0x3b>;

 	lvds_ser4: serializer@77 {
-		compatible = "ti,ds90ub913aq";
+		compatible = "ti,crapub913aq";
 		reg = <0x58>;
 		slave-mode;
 	};
@@ -252,7 +258,7 @@
 		 <0x30 0x3c>;

 	lvds_ser5: serializer@78 {
-		compatible = "ti,ds90ub913aq";
+		compatible = "ti,crapub913aq";
 		reg = <0x58>;
 		slave-mode;
 	};
@@ -276,7 +282,7 @@
 		 <0x30 0x3d>;

 	lvds_ser6: serializer@79 {
-		compatible = "ti,ds90ub913aq";
+		compatible = "ti,crapub913aq";
 		reg = <0x58>;
 		slave-mode;
 	};
